Source/Core/Plugins/EventProcessor/Default/03_ManualStackingPlugin.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Exceptionless.Core.Pipeline;
+using Exceptionless.Core.Extensions;
+using Exceptionless.Core.Models;
+
+namespace Exceptionless.Core.Plugins.EventProcessor {
+    [Priority(3)]
+    public class ManualStackingPlugin : EventProcessorPluginBase {
+        public override Task EventProcessingAsync(EventContext context) {
+            var data = context.Event.Data;
+            if (!data.ContainsKey(Event.KnownDataKeys.ManualStackingKey)
+                || data[Event.KnownDataKeys.ManualStackingKey] == null)
+                return Task.CompletedTask;
+
+            context.StackSignatureData.AddItemIfNotEmpty(nameof(Event.KnownDataKeys.ManualStackingKey), data[Event.KnownDataKeys.ManualStackingKey].ToString());
```

Comment:
We can make this whatever we want it to be. I am trying to make this clean and simple all the way down to the raw HTTP level. So if I do a CURL command to post data, I don't want to use @manual_stacking_key, it's just too verbose and I think @stack is still pretty self explanatory.

As for the StackSignatureData, we can add it as whatever we want. So I'd say something like ManualStackingKey is good there.